src/ResultsActivity/ResultsActivityContainer.js

```diff
@@ -123,6 +125,10 @@ class ResultsActivityContainer extends Component {
     )
   }
 
+  update() {
+    this.getActivityList()
+  }
```

Comment:
Put this in its own update to keep consistency with the other pages, plus we can add sorting behaviour on refresh here in the future.